app/src/ble.c

```diff
@@ -548,6 +548,28 @@ static void zmk_ble_ready(int err) {
     update_advertising();
 }
 
+int zmk_ble_reset_clear_all_bonds() {
+    LOG_WRN("Clearing all existing BLE bond information from the keyboard");
+
+    for (int i = 0; i < 10; i++) {
```

Comment:
This really isn't necessary, this was a holdover from previously using multiple "IDs" for profiles. Can just be `bt_unpair(BT_ID_DEFAULT, null)` without the loop.